tests/utils/dockercli/volumelifecycle.go

```diff
@@ -85,7 +90,21 @@ func ReadFromVolume(ip, volName, containerName, fileName string) (string, error)
 // DeleteVolume helper deletes the created volume as per passed volume name.
 func DeleteVolume(ip, name string) (string, error) {
 	log.Printf("Destroying volume [%s]\n", name)
-	return ssh.InvokeCommand(ip, dockercli.RemoveVolume+name)
+	var out string
+	var err error
+	for attempt := 0; attempt <= maxAttempt; attempt++ {
```

Comment:
if you are planning to set 30seconds max then suppose to start with `attempt=1` or `attempt < maxAttempt`